DiscordBot.Modules/CommandModules/ReactionModule.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Linq;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using Discord;
+using Discord.Commands;
+using Microsoft.Extensions.Logging;
+
+namespace DiscordBot.Modules.CommandModules
+{
+    public class ReactionModule : ModuleBase
+    {
+        private readonly ILogger<PingModule> _logger;
+
+        public ReactionModule(ILogger<PingModule> logger)
+        {
+            _logger = logger;
+        }
+
+        [Command("react")]
+        [Summary("Der Bot reagiert mit dem angegebenen Emoji auf die angegebene Nachricht.")]
+        public async Task AddReaction(string name, string URL)
+        {
+            await ReactWithEmoji(name, URL);
+        }
+        private async Task ReactWithEmoji(string name, string URL)
+        {
+            string emojilist = name switch
+            {
+                "sans" => "<a:sans:719488632104288286>",
+                "thonk" => "<:thonk:718853430969368637>",
+                "mindblowing" => "<:mindblowing:719492411369324584>",
+                "rotatethink" => "<a:rotatethonk:719491269885427752>",
+                "hyper" => "<a:hyper:719494595313926244>",
+                "xd" => "<:xd:719510255402352690>",
+                "rainbowcat" => "<a:catrainbow1:719513824511656037><a:catrainbow2:719513824553861121>\n<a:catrainbow3:719513824905920603><a:catrainbow4:719513825040400415>",
```

Comment:
Why would you match this name when you return an error anyways?

Reply:
Good Question, not working on the bot this time